### PR TITLE
Fix typos in protocol_oriented_generics.ipynb

### PR DESCRIPTION
**Pull request by prayash**
Inheritance was spelled incorrectly in a few places.

### CODE REVIEW

**Comment by prayash** [comment]
@googlebot I signed it!

File: docs/site/tutorials/protocol_oriented_generics.ipynb [hunk 1]
```diff
@@ -75,9 +75,9 @@
    "source": [
     "## Protocols\n",
     "\n",
-    "Inheritence is a powerful way to organize code in programming languages that allows you to share code between multiple components of the program.\n",
+    "Inheritance is a powerful way to organize code in programming languages that allows you to share code between multiple components of the program.\n",
     "\n",
-    "In Swift, there are different ways to express inheritence. One of those ways you may be familiar with already - class inheritence. However, Swift has another way - protocols.\n",
+    "In Swift, there are different ways to express inheritance. One of those ways you may be familiar with already - class inheritance. However, Swift has another way - protocols.\n",
```
Comment (dan-zheng) [inline docs/site/tutorials/protocol_oriented_generics.ipynb]: Some unrelated suggestions:
```suggestion
    "In Swift, there are different ways to express inheritance. You may be already be familiar with one of those ways, from other languages: class inheritance. However, Swift has another way: protocols.\n",
```

Comment (BradLarson) [reply docs/site/tutorials/protocol_oriented_generics.ipynb]: I think you may have an extra "be" in the suggestion: "You may already be familiar..."

Comment (dan-zheng) [reply docs/site/tutorials/protocol_oriented_generics.ipynb]: Thanks for noticing! Fixed the suggestion.

**Comment by dan-zheng** [comment]
@prayash: could you please address this suggestion? https://github.com/tensorflow/swift/pull/326#discussion_r349729836
I don't have permission to merge it myself, for some reason.

**Comment by prayash** [comment]
@dan-zheng Done!

**Comment by dan-zheng** [comment]
Thanks @prayash!

**Pull request merged**